tutorials/incompressible_flow_model.ipynb

```diff
@@ -595,7 +595,10 @@
    "id": "5ae70b03-3dde-4413-a62d-e3d8586182ce",
    "metadata": {},
    "source": [
-    "Unsurprisingly, the resulting pressure solution closely resembles that obtained using the parent class `UnitaryFractureSource`, with an additional high-pressure region around the point $(x,y) = (0.7, 0.7)$."
+    "Unsurprisingly, the resulting pressure solution closely resembles that obtained using the parent class `UnitaryFractureSource`, with an additional high-pressure region around the point $(x,y) = (0.7, 0.7)$.\n",
+    "\n",
+    "## What have we explored\n",
+    "The concept of a `Model` has been discussed. The mixed-dimensional scalar flow problem was defined, and with some minor adjustments, complex scenarios could be approximated. Having made no assumptions about the ambient dimension, the extension to a three-dimensional setting consists only of creating a mixed-dimensional mesh and defining boundary values.\n"
```

Comment:
```suggestion
    "The concept of a `Model` has been discussed. The mixed-dimensional scalar flow problem was defined, and with some minor adjustments, complex scenarios could be approximated. Having made no assumptions about the ambient dimension, the extension to a three-dimensional setting consists only of creating a three-dimensinal mixed-dimensional grid instead of the two-dimensional used above.\n"
```